packages/inter-protocol/src/proposals/price-feed-proposal.js

```diff
@@ -109,7 +109,7 @@ export const createPriceFeed = async (
   ]);
 
   /** @type {import('@agoric/zoe/src/contracts/priceAggregator.js').PriceAggregatorContract['terms']} */
-  const terms = await deeplyFulfilled(
+  const terms = await deeplyFulfilledObject(
```

Comment:
Lint is reminding you here that the import wasn't updated in htis file.

Reply:
I do not see any current lint warning, so resolving.